lib/spire/item.rb

```diff
@@ -210,8 +210,17 @@ def find(id, params = {})
       # @raise [Spire::Error] if the item could not be found.
       #
       # @return [Spire::Item]
-      def search(query)
-        client.find_many(Spire::Item, "/inventory/items/", { q: query })
+      def search(query, limit = 10)
```

Comment:
@Dujota I feel like the default should `nil` and the implementation would be something like this

```
def search(query, limit = nil)
  options = { q: query }
  options[:limit] = limit if limit
  client.find_many(Spire::Item, "/inventory/items/", options}
end
```

Otherwise, I think someone who uses this method might get confused as to why it's not returning more results than they expect.

Reply:
updates pushed to PR